tests/rdma/run.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/python3
+"""
+==========================================================================
+run.py - script for test client for Valkey Over RDMA (Linux only)
+         note that is script need root privilege.
+--------------------------------------------------------------------------
+Copyright (C) 2024  zhenwei pi <pizhenwei@bytedance.com>
+
+This work is licensed under BSD 3-Clause, License 1 of the COPYING file in
+the top-level directory.
+==========================================================================
+"""
+import os
+import subprocess
+import netifaces
+import time
+
+def build_program():
+    cmd = "gcc tests/rdma/rdma-test.c -o tests/rdma/tmp/valkey-rdma-test -lrdmacm -libverbs -lpthread -g"
+    p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+    if p.wait():
+        print("Valkey Over RDMA build valkey-rdma-test [FAILED]")
+        os._exit(1);
+
+    print("Valkey Over RDMA build valkey-rdma-test <tests/rdma/tmp/valkey-rdma-test> [OK]")
```

Comment:
If we change the binary name, then update these messages too.

An idea, if you want a correct path in the message:

```suggestion
    dir = dirname(__file__)
    print("Valkey Over RDMA build %s/rdma-test [OK]" % dir)
```

Reply:
This is removed from `run.py`, use a separated `Makefile` instead.